README.md

```diff
@@ -446,6 +464,7 @@ const main = async () => {
 | `put`     | Key has been updated        | `key, value` (any)   |
 | `del`     | Key has been deleted        | `key` (any)          |
 | `batch`   | Batch has executed          | `operations` (array) |
+| `clear`   | Entries were deleted        | `options` (object)   |
```

Comment:
Not 100% sure about the argument (`options`). I figured a listener might want to inspect the range options, to find out what was deleted. But that information is not exact (in contrast with the `put`, `del` and `batch` events that tell you exactly which keys were modified).